.travis.yml

```diff
@@ -175,6 +175,7 @@ matrix:
           - qt512x11extras
           - qt512serialbus
     install:
+      - pip3 install --upgrade pip setuptools
```

Comment:
Is this necessary? :)

Reply:
The pipeline tests were failing because of an outdated setuptools. After some research and reading it was clear setuptools needed to be updated. Adding this immediately fixed the issue.

Reply:
Strange... Because my other PR ran just fine without this. But it's not a bad idea to update some these :-)